ucs/clientlist.cpp

```diff
@@ -992,7 +1031,7 @@ int Client::ChannelCount() {
 
 }
 
-void Client::JoinChannels(std::string ChannelNameList) {
+void Client::JoinChannels(std::string ChannelNameList, bool command_directed) {
```

Comment:
Should this be `const std::string&` to avoid making a copy? Also, `channel_name_list`?